README.md

```diff
@@ -133,6 +133,67 @@ enum Shape {
 }
 ```
 
+## Switch Statements
+1. Enum cases with an associated value should have a appropriate label as opposed to just types
```

Comment:
This sentence needs a `.` at the end 😉 .

Also, what is the rationale for this recommendation? It should be added as well.